### PR TITLE
Add license and requirements

### PR DESCRIPTION
**Pull request by nvnieuwk**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+pandas==1.1.5
+pybedtools==0.9.0
+pysam==0.20.0
+python-dateutil==2.8.2
+pytz==2022.7.1
+s3transfer==0.6.0
+scikit-learn==1.0.2
+scipy==1.7.3
+six==1.16.0
+-e git+https://github.com/CenterForMedicalGeneticsGhent/svtk@52270d1ee5f50edd0d8f8c77ae11c8a788318b52#egg=svtk
```
Comment (matthdsm) [inline requirements.txt]: What's this do?

Comment (nvnieuwk) [reply requirements.txt]: I don't really know. This file is the result of `pip freeze > requirements.txt`

Comment (matthdsm) [reply requirements.txt]: requirements are defined in `setup.py` so I'm not sure this is needed

Comment (nvnieuwk) [reply requirements.txt]: I know but when running `pip install -e .` it gave me errors that pysam and cython weren't installed (which are both imported in setup.py). That the reason I added it there (just an easy step instead having to install those separately)

**Review by matthdsm — COMMENTED** [review]
I'd also pin minimal versions (where applicable) instead of a full pin

**Review by matthdsm — APPROVED** [review]

**Pull request merged**